tests/integrations/raftstore/test_transfer_leader.rs

```diff
@@ -110,12 +110,73 @@ fn test_pd_transfer_leader<T: Simulator>(cluster: &mut Cluster<T>) {
     }
 }
 
+fn test_pd_transfer_leader_multi_target<T: Simulator>(cluster: &mut Cluster<T>) {
+    let pd_client = Arc::clone(&cluster.pd_client);
+    pd_client.disable_default_operator();
+
+    cluster.run();
+
+    cluster.must_put(b"k1", b"v1");
+    cluster.add_send_filter(IsolationFilterFactory::new(3));
+    cluster.must_put(b"k2", b"v2");
+    // append index of the cluster will finally be 2, 2, 1
```

Comment:
How about using `must_get_equal` and `must_get_none` ensure that before transfer leader.